.fixtures.yml

```diff
@@ -0,0 +1,7 @@
+fixtures:
+  repositories:
+    stdlib:
+      repo: https://github.com/puppetlabs/puppetlabs-stdlib.git
+      ref: 4.25.0
```

Comment:
can you please remove this line? we always want to run unit tests against the master branch to detect new issues early. acceptance tests use the latest release

Reply:
I removed the `ref` line.